dex/src/main/scala/com/wavesplatform/dex/db/DbKeys.scala

```diff
@@ -130,13 +130,19 @@ object DbKeys {
         bb.get(name)
         val decimals = bb.getInt
         val hasScript = bb.get == 1
+        val isNft = Try(bb.get).fold(_ => false, _ == 1)
```

Comment:
we should add a comment here about backward compatibility. sth like "Try here is needed for backward compatibility"